DigitalLearningSolutions.Data.Tests/DataServices/RegistrationDataServiceTests.cs

```diff
@@ -167,10 +177,14 @@ public void Sets_notification_preferences_correctly_on_centre_manager_admin_regi
             using var transactionScope = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled);
 
             // Given
-            var registrationModel = RegistrationModelTestHelper.GetDefaultCentreManagerRegistrationModel();
+            var registrationModel =
+                RegistrationModelTestHelper.GetDefaultCentreManagerRegistrationModel(
+                    email: "bn.uhrawol@sk",
+                    categoryId: 1
+                );
 
             // When
-            service.RegisterAdmin(registrationModel);
+            service.RegisterAdmin(registrationModel, 4046);
 
             // Then
             var user = userDataService.GetAdminUserByEmailAddress(registrationModel.PrimaryEmail)!;

```

Comment:
Does this definitely get the user you've just added? You're not longer setting the email address, so I'm a bit suspicious.

Reply:
Similar in other tests